MatrixKit/Models/Room/MXKRoomBubbleCellData.h

```diff
@@ -65,6 +65,11 @@ typedef enum : NSUInteger {
 @property (nonatomic, readonly) MXSession *mxSession;
 
 /**
+ The room state
+ */
+@property (nonatomic, readonly) MXRoomState *mxRoomState;
```

Comment:
I suggest using the current room state instead of store a snapshot of the room state in each cell data instance.